core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2354,7 +2354,7 @@ private[spark] object Utils extends Logging {
    * A spark url (`spark://host:port`) is a special URI that its scheme is `spark` and only contains
    * host and port.
    *
-   * @throws SparkException if `sparkUrl` is invalid.
+   * @note Throws `SparkException` if sparkUrl is invalid.
```

Comment:
`SparkException` reference is not found in javadoc.

```
[error] .../java/org/apache/spark/util/Utils.java:841: error: reference not found
[error]    * @throws SparkException if sparkUrl is invalid.
[error]      ^
```

I am not too sure using `@note` instead is right.